src/Kros.AspNetCore/Middlewares/RoutePatternMiddleware.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Kros.AspNetCore.Middlewares
+{
+    /// <summary>
+    /// Middleware which adds route pattern claim.
+    /// </summary>
+    public class RoutePatternMiddleware
+    {
+        private const string RoutePatternClaimType = "route_pattern";
```

Comment:
Duplicate constant. You can create it as `internal` and use in `RoutePatternInitializer`.

Reply:
Constants are in different projects. Projects have not relationships with each other.